lib/src/_shared/PickerBase.d.ts

```diff
@@ -5,4 +5,5 @@ export interface PickerBaseProps {
   onChange: (date: MaterialUiPickersDate) => void;
   autoOk?: boolean;
   ampm?: boolean;
+  pickerRef:(e:Element)=>void;
```

Comment:
`pickerRef` should be optional, shouldn't it?

Also, typescript validation failed, here's main part of error logs:

`Property 'pickerRef' is missing in type '{ keyboard: true; mask: (string | RegExp)[]; placeholder: string; value: Date; onChange: (date: D..`

Reply:
Oh yes sorry, i forgot the "?", i'll fix it